src/ui/widgets/EmbeddedDisplay/embeddedDisplay.tsx

```diff
@@ -40,41 +41,56 @@ export const EmbeddedDisplay = (
 
   log.debug(description);
 
-  // Apply the height to the top level if relative layout and none have been provided
-  if (props.position instanceof RelativePosition) {
-    props.position.height = props.position.height || description.height;
-    props.position.width = props.position.width || description.width;
-  }
+  // Check whether to override OPI autoZoomToFit
+  const autoZoomToFitOverride =
+    typeof props.autoZoomToFitOverride === "undefined"
+      ? true
+      : props.autoZoomToFitOverride;
 
-  // Apply the height to the top level if relative layout and none have been provided
-  if (props.position instanceof RelativePosition) {
-    props.position.height = props.position.height || description.height;
-    props.position.width = props.position.width || description.width;
-  }
+  // Get the screen height
+  const heightString =
+    typeof description.position.height === "undefined"
+      ? "10"
```

Comment:
Why the number 10 in particular here?

Reply:
This was random. The height is always defined in the opi file so this shouldn't be a problem, but I have updated it so that the default should it not be set is the window height/width.